ts/node/logs.ts

```diff
@@ -1,13 +1,8 @@
 import { ipcRenderer } from 'electron';
+import pTimeout from 'p-timeout';
 
-export async function deleteAllLogs() {
-  return new Promise((resolve, reject) => {
-    ipcRenderer.once('delete-all-logs-complete', resolve);
+import { DURATION } from '../session/constants';
 
-    setTimeout(() => {
-      reject(new Error('Request to delete all logs timed out'));
-    }, 5000);
-
-    ipcRenderer.send('delete-all-logs');
-  });
+export function deleteAllLogs(): Promise<void> {
+  return pTimeout(ipcRenderer.invoke('delete-all-logs', false), 5 * DURATION.SECONDS);
```

Comment:
Why pTimeout?

Reply:
signal uses it, and I figured we should probably use it more too :D